Examples/Scripts/7T/IcaFixProcessingBatch.7T.sh

```diff
@@ -129,32 +129,33 @@ main() {
 	# validate environment variables
 	# validate_environment_vars $@
 
-	# establish queue for job submission
-	QUEUE="-q hcp_priority.q"
+	#NOTE: syntax for QUEUE has changed compared to earlier pipeline releases,
+	#DO NOT include "-q " at the beginning
+	#default to no queue, implying run local
+	QUEUE=""
+	#QUEUE="hcp_priority.q"
 
 	# establish list of conditions on which to run ICA+FIX
-	CondList=""
-	CondList="${CondList} rfMRI_REST1_7T"
-	CondList="${CondList} rfMRI_REST2_7T"
-	CondList="${CondList} rfMRI_REST3_7T"
-	CondList="${CondList} rfMRI_REST4_7T"
+	CondList=()
+	CondList+=(rfMRI_REST1_7T)
+	CondList+=(rfMRI_REST2_7T)
+	CondList+=(rfMRI_REST3_7T)
+	CondList+=(rfMRI_REST4_7T)
 
 	# establish list of directions on which to run ICA+FIX
-	DirectionList=""
-	DirectionList="${DirectionList} PA"
-	DirectionList="${DirectionList} AP"
-	DirectionList="${DirectionList} PA"
-	DirectionList="${DirectionList} AP"
+	DirectionList=()
+	DirectionList+=(PA)
+	DirectionList+=(AP)
```

Comment:
If `CondList` contains 4 runs, doesn't `DirectionList` need to contain 4 values as well?

Reply:
With the for loop below, having 4 didn't make sense.  Since this is only used in the input filenames, then it should just be included in the run names array to simplify things.

Reply:
Right.  A single `for` loop, with the task names and polarities combined seems like the way to go (and consistent with how other batch scripts are set up).